### ISSUE TITLE
Set On-Device Folders for Markdown Notes 

### ISSUE THREAD

**Issue by BlackberrieCake137** [ismartcoding/plain-app#93]
**Is your feature request related to a problem? Please describe.**

Notes as-is arent set to browse folders on the system at all, unlike audio and file view. This makes it aggravating if you have markdown files in specific directories (like a synced Obsidian vault)

**Describe the solution you'd like**
Adding a folder browser and pinned folders to the notes view so you can set folders to quickly access note folders from. A function to add folders here and the ability to move notes would also be really nice. 

**Describe alternatives you've considered**

**Additional context**
The way Obsidian handles folders of markdown notes in 'vaults' works really well. Obsidian is obviously proprietary but its way of handling this is simple.

**Comment by readandprogress** [comment]
I agree! One of the few features that this app lacks, It will also make the notes way more portable, in a way is already there, you can edit markdown files in the files in the files section but you can't preview them so adding this or previews for markdown files would be awesome! 

Keep at the great work, Love your app!

**Comment by KyrillGobber** [comment]
I'm looking for exactly this. All my notes are in markdown and Isync them between my devices. Plainapp has a perfect editor already imho, and I'd love to set the notes folder to my synced one
